### PR TITLE
Feat(Signing): Message Signing

### PR DESCRIPTION
**Pull request by nduchak**

### CODE REVIEW

**Comment by nduchak** [comment]
Resolve https://github.com/aeternity/aepp-sdk-js/issues/904

**Comment by nduchak** [comment]
Resolve #902, #899,  aeternity/tipwallet#87

File: es/utils/crypto.js [hunk 1]
```diff
@@ -359,18 +359,18 @@ export function prepareTx (signature, data) {
 }
 
 export function personalMessageToBinary (message) {
-  const p = Buffer.from('æternity Signed Message:\n', 'utf8')
+  const p = Buffer.from('aeternity Signed Message:\n', 'utf8')
```
Comment (davidyuk) [inline es/utils/crypto.js]: what is the problem with æ? As I know, we are using it on Ledger

Comment (mradkov) [reply es/utils/crypto.js]: Hans suggested we use “ae” instead as otherwise we can face problems in the future.

**Pull request merged**